### ISSUE TITLE
Ditto Clipboard opening in background behind every other window

### ISSUE THREAD

**Issue by andytham** [kangyu-california/PersistentWindows#92]
**Issue:** Calling the hotkey to show the 'quick paste' function puts the application window in the background.
**Version:** 5.17 and 5.18
**Screenshot:** 
![image](https://user-images.githubusercontent.com/33642114/110008580-e3712780-7ce9-11eb-9a66-3aa4f86cf181.png)
[URL to Ditto Clipboard](https://ditto-cp.sourceforge.io/)

It's a system tray application so that might be related. It should open an application window at the cursor when global hotkey is called.

Issue is not present when I turn off 'auto restore' or exit PersistentWindow.

**Comment by kangyu-california** [comment]
When activate ditto window, PW tries to move it back to previous open position, which is against the purpose of Ditto to pop up as close as possible to current application where Ditto is summoned.

To disable PW from auto restore ditto window, please try release 5.19, user need to explicitly "instruct" PW to exclude a window from auto restore, this is achieved by holding Ctrl+Shift keys then move the Ditto window.

Hope this helps.

**Comment by andytham** [comment]
Thank you for the update but it doesn't seem to help, Ditto is still hidden behind every other app and is impossible to see for the most part.
I'll keep auto-restore off for now.

**Comment by kangyu-california** [comment]
I guess you are running PW with -fix_zorder option, which ironically does not preserve the zorder of Ditto window somehow.
The issue is expected, that is why this option is by default off.

For user who would like to restore zorder of previous window layout, it is suggested to use snapshot feature instead.

**Comment by kangyu-california** [comment]
Can't reproduce on my PC.
What puzzles me is why pause auto-restore would help.  
my understanding is activating Ditto should not trigger overall auto restore, especially true with Ctrl+Shift moving trick.

Posting most recent event log for event 9999 + 9990 would definitely help, thanks!

**Comment by andytham** [comment]
Is this correct?

9999
```Log Name:      Application
Source:        Application
Date:          3/9/21 15:00:16
Event ID:      9999
Task Category: None
Level:         Information
Keywords:      Classic
User:          N/A
Computer:      REDEMPTION
Description:
The description for Event ID 9999 from source Application cannot be found. Either the component that raises this event is not installed on your local computer or the installation is corrupted. You can install or repair the component on the local computer.

If the event originated on another computer, the display information had to be saved with the event.

The following information was included with the event: 

PersistentWindows: no qualified position data to restore minimized window "Event Viewer"

The message resource is present but the message was not found in the message table

Event Xml:
<Event xmlns="http://schemas.microsoft.com/win/2004/08/events/event">
  <System>
    <Provider Name="Application" />
    <EventID Qualifiers="0">9999</EventID>
    <Level>4</Level>
    <Task>0</Task>
    <Keywords>0x80000000000000</Keywords>
    <TimeCreated SystemTime="2021-03-09T20:00:16.383330600Z" />
    <EventRecordID>861429</EventRecordID>
    <Channel>Application</Channel>
    <Computer>REDEMPTION</Computer>
    <Security />
  </System>
  <EventData>
    <Data>PersistentWindows: no qualified position data to restore minimized window "Event Viewer"</Data>
  </EventData>
</Event>
```

9990
```Log Name:      Application
Source:        Application
Date:          3/9/21 15:04:41
Event ID:      9990
Task Category: None
Level:         Information
Keywords:      Classic
User:          N/A
Computer:      REDEMPTION
Description:
The description for Event ID 9990 from source Application cannot be found. Either the component that raises this event is not installed on your local computer or the installation is corrupted. You can install or repair the component on the local computer.

If the event originated on another computer, the display information had to be saved with the event.

The following information was included with the event: 

PersistentWindows: ignore window Ditto

The message resource is present but the message was not found in the message table

Event Xml:
<Event xmlns="http://schemas.microsoft.com/win/2004/08/events/event">
  <System>
    <Provider Name="Application" />
    <EventID Qualifiers="0">9990</EventID>
    <Level>4</Level>
    <Task>0</Task>
    <Keywords>0x80000000000000</Keywords>
    <TimeCreated SystemTime="2021-03-09T20:04:41.254959100Z" />
    <EventRecordID>861430</EventRecordID>
    <Channel>Application</Channel>
    <Computer>REDEMPTION</Computer>
    <Security />
  </System>
  <EventData>
    <Data>PersistentWindows: ignore window Ditto</Data>
  </EventData>
</Event>```

**Comment by andytham** [comment]
Hmm that's not quite the behavior I wanted, I only want the Ditto window to show when press this hotkey 
![image](https://user-images.githubusercontent.com/33642114/110536340-59f99500-80ef-11eb-856e-4f60db1e98dd.png)
and deactivate as soon as I'm done or click off it.
But "Always on top" just leaves it on my screen and I have to disable "always on top" or 'X' out of it. (am on latest build too)

I might have a 3rd party program lurking somewhere so I'll look into it some more since you can't replicate. Thanks for the help.

**Comment by kangyu-california** [comment]
PW use Ctrl+Win hotkey to send a newly activated window to background, as is documented in Help. 

That setting has conflict with your hot key assignment.
To workaround this issue, use default Ctrl+` to activate Ditto. Or in your case, remove the tick on Win key.

**Comment by kangyu-california** [comment]
In future PW release I will add command line option to disable Ctrl + Win hotkey feature

**Comment by andytham** [comment]
Yep that did it, but I feel like it shouldn't have affected it since my hotkey also requires ALT.

Thank you for your work!

**Comment by kangyu-california** [comment]
Fixed in 5.20.
Ctrl+Win hot key is no longer used for most Apps except for RemoteDesktop
